flutter_cache_manager/pubspec.yaml

```
@@ -16,7 +16,7 @@ dependencies:
   sqflite: "^1.1.7+2"
   pedantic: "^1.8.0+1"
   clock: ^1.0.1
-  file: ^5.1.0
+  file: ^6.0.0-nullsafety.1
```

Is the this only relevant part of the commit? A lot of other files are changed as well.

Why not support both 5.1 and 6 for people who are not on master? Are there breaking changes?
```suggestion
  file: '>=5.1.0 <7.0.0'
```